.github/workflows/release.yml

```diff
@@ -11,9 +11,9 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
     - name: Use Node.js
-      uses: actions/setup-node@v2
+      uses: actions/setup-node@v3
```

Comment:
Why are these necessary for yarn pnp?

Reply:
Not necessary, but it is good to use latest actions?

Reply:
Yes, it can happen in a different PR.

I think the changes in this PR are critical enough that the changes should be minimal and deliberate as possible. Easier to be aware and review all the changes happening.

I think things like environmental changes are best to do it in isolation. Better documented, easier to trace back if problematic, etc.